### PR TITLE
improving_docs - No functionality added

### PR DESCRIPTION
**Pull request by jposada202020**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -56,19 +56,17 @@ To install in a virtual environment in your current project:
...
-.. code-block:: python
+.. code-block:: python3
 
     import time
     import board
-    import busio
     import adafruit_lis2mdl
 
-    i2c = busio.I2C(board.SCL, board.SDA)
+    i2c = board.I2C()
```
Comment (kattni) [inline README.rst]: Should this include the `uses board.SCL and board.SDA` comment you included with the others?

File: adafruit_lis2mdl.py [hunk 2]
```diff
@@ -91,7 +89,31 @@ class LIS2MDL:  # pylint: disable=too-many-instance-attributes
...
+        .. code-block:: python
+
+            import board
+            import adafruit_lis2mdl
+
+        Once this is done you can define your `board.I2C` object and define your sensor object
+
+        .. code-block:: python
+
+            i2c = board.I2C()
```
Comment (kattni) [inline adafruit_lis2mdl.py]: Same question.

File: examples/lis2mdl_calibrate.py [hunk 3]
```diff
@@ -5,10 +5,9 @@
 
 import time
 import board
-import busio
 import adafruit_lis2mdl
 
-i2c = busio.I2C(board.SCL, board.SDA)
+i2c = board.I2C()
```
Comment (kattni) [inline examples/lis2mdl_calibrate.py]: Same question.

File: examples/lis2mdl_interrupt.py [hunk 4]
```diff
@@ -3,10 +3,9 @@
 
 import time
 import board
-import busio
 import adafruit_lis2mdl
 
-i2c = busio.I2C(board.SCL, board.SDA)
+i2c = board.I2C()
```
Comment (kattni) [inline examples/lis2mdl_interrupt.py]: Same question.

File: examples/lis2mdl_simpletest.py [hunk 5]
```diff
@@ -5,10 +5,9 @@
 
 import time
 import board
-import busio
 import adafruit_lis2mdl
 
-i2c = busio.I2C(board.SCL, board.SDA)
+i2c = board.I2C()
```
Comment (kattni) [inline examples/lis2mdl_simpletest.py]: Same question.

**Review by kattni — CHANGES_REQUESTED** [review]
Thanks for the update! I have one question below, that applies to multiple files.

File: README.rst (hunk 1 above)
Comment (jposada202020) [reply README.rst]: @kattni  I could included here, in the newer libraries, meaning they just create the object without the comment so I tried to keep the same style as the library, no problem to add it, let me know

**Comment by jposada202020** [comment]
@kattni Changes were introduced in https://github.com/adafruit/Adafruit_CircuitPython_LIS2MDL/pull/13/commits/a5a8dfb3113f35f419452c2715b7ff99037e39c4.

Thanks.

File: examples/lis2mdl_simpletest.py (hunk 5 above)
Comment (jposada202020) [reply examples/lis2mdl_simpletest.py]: Changes in https://github.com/adafruit/Adafruit_CircuitPython_LIS2MDL/pull/13/commits/a5a8dfb3113f35f419452c2715b7ff99037e39c4

File: examples/lis2mdl_interrupt.py (hunk 4 above)
Comment (jposada202020) [reply examples/lis2mdl_interrupt.py]: Changes in https://github.com/adafruit/Adafruit_CircuitPython_LIS2MDL/pull/13/commits/a5a8dfb3113f35f419452c2715b7ff99037e39c4

File: examples/lis2mdl_calibrate.py (hunk 3 above)
Comment (jposada202020) [reply examples/lis2mdl_calibrate.py]: Changes in https://github.com/adafruit/Adafruit_CircuitPython_LIS2MDL/pull/13/commits/a5a8dfb3113f35f419452c2715b7ff99037e39c4

File: adafruit_lis2mdl.py (hunk 2 above)
Comment (jposada202020) [reply adafruit_lis2mdl.py]: Changes in https://github.com/adafruit/Adafruit_CircuitPython_LIS2MDL/pull/13/commits/a5a8dfb3113f35f419452c2715b7ff99037e39c4

File: README.rst (hunk 1 above)
Comment (jposada202020) [reply README.rst]: Changes in https://github.com/adafruit/Adafruit_CircuitPython_LIS2MDL/pull/13/commits/a5a8dfb3113f35f419452c2715b7ff99037e39c4

**Review by kattni — APPROVED** [review]

**Pull request merged**